go.mod

```diff
@@ -252,6 +266,3 @@ require (
 	rsc.io/tmplfunc v0.0.3 // indirect
 	sigs.k8s.io/yaml v1.4.0 // indirect
 )
-
-// See https://github.com/prysmaticlabs/grpc-gateway/issues/2
```

Comment:
As described in https://github.com/datachainlab/ethereum-ibc-relay-prover/pull/38#discussion_r2050349142, this replace directive causes a build failures due to mismatch of signatures.